api/auth/validate_oauth.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2023 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package auth
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/go-vela/server/scm"
+	"github.com/go-vela/server/util"
+)
+
+// swagger:operation GET /validate-oauth authenticate ValidateOAuthToken
+//
+// Validate that a user oauth token was created by Vela
+//
+// ---
+// produces:
+// - application/json
+// parameters:
+// - in: header
+//   name: Token
+//   type: string
+//   required: true
+//   description: >
+//     OAuth integration user access token
+// responses:
+//   '200':
+//     description: Successfully validated
+//     schema:
+//       "$ref": "#/definitions/Token"
+//   '401':
+//     description: Unable to validate
+//     schema:
+//       "$ref": "#/definitions/Error"
+//   '503':
+//     description: Service unavailable
+//     schema:
+//       "$ref": "#/definitions/Error"
```

Comment:
i don't think we return 503 on this endpoint?

Reply:
oh good point, copy pasta. ill remove